content/adr/adopt-semantic-versioning.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Adopt semantic versioning for releases"
+date: "2022-04-13"
+decision: "We will adopt semantic versioning for naming our releases"
+status: "accepted"
+categories:
+  - "Processes"
```

Comment:
@hussainweb not sure why this is not picking up in the catgories in main page. Is there somewhere else we need to add?

Reply:
It should, based on my understanding of the templates. Let's merge this and we can look into that.

Reply:
ok

Reply:
Now it is not showing in All as well. Not sure if due to changed as Proposed.

https://deploy-preview-235--axelerant-engg-handbook.netlify.app/adr/adopt-semantic-versioning/